### ISSUE TITLE
Support long-running HTTP calls in AWS

### ISSUE THREAD

**Issue by jonathanlukas** [camunda/connectors#2049]
__Is your feature request related to a problem? Please describe.__

We are currently using the connector runtime in a hybrid setup to perform long-running (10-60min) calls to a service.

Right now, the AWS NAT Gateway closes the connection after 350 seconds idle.

__Describe the solution you'd like__

A way to overcome this is described here: https://aws.amazon.com/blogs/networking-and-content-delivery/implementing-long-running-tcp-connections-within-vpc-networking/

TL;DR: The host system needs to configure the tcp socket to have a relatively short keepAlive ping.

The application opening the tcp connection (to perform an http request) needs to tell the socket that `SO_KEEPALIVE` should be applied.

__Describe alternatives you've considered__

Not use a connector OR implement my own.

__Additional context__

no
